src/Game.java

```diff
@@ -0,0 +1,115 @@
+import java.util.*;
+
+public class Game {
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+    private static final int MAX_ERRORS = 8;
+
+    private Scanner scanner = new Scanner(System.in);
+
+    private String wordToFind;
```

Comment:
Is this word that user is trying to guess? If so, why isnt it named `wordToGuess`

Reply:
Yes, I changed.